CC/src/AutoSegmentationTools.cpp

```diff
@@ -129,8 +129,8 @@ bool AutoSegmentationTools::extractConnectedComponents(GenericIndexedCloudPersis
 }
 
 bool AutoSegmentationTools::frontPropagationBasedSegmentation(	GenericIndexedCloudPersist* theCloud,
-																PointCoordinateType radius,
 																ScalarType minSeedDist,
+																PointCoordinateType radius,
```

Comment:
Nice catch! Luckily this method is not used anymore.

Out of consistency with what is generally done in CC, I would have kept the radius before the seed parameter (so I would have inverted them in the header file and not in the source file).

Reply:
I was expecting this to be the case. I intended to keep the interface intact to avoid compatibility issues, but consistency is sound argument.